doc/guide-meson.build

```diff
@@ -56,6 +56,7 @@ custom_target(
 )
 
 
+if prog_fop.found()
 fop = custom_target(
```

Comment:
Is this hunk really needed?  If yes, could you indent it, please?

Reply:
Indented.